src/main/groovy/com/cloudbees/groovy/cps/CpsTransformer.groovy

```diff
@@ -78,7 +79,7 @@ import static org.codehaus.groovy.syntax.Types.*
  * @author Kohsuke Kawaguchi
  */
 class CpsTransformer extends CompilationCustomizer implements GroovyCodeVisitor {
-    private int iota=0;
+    private static final AtomicLong iota = new AtomicLong();
```

Comment:
Well compilation is certainly not cheap to begin with. One `AtomicLong` access per method is hardly going to matter.